source/projects/jungle_beat/iteration_1.md

```diff
@@ -1,27 +1,24 @@
-# Iteration 1 - Append, All and Count (Single Node)
+# Iteration 1 - Append, All and Count
 
-Great! We have nodes. Next step is to build out the `JungleBeat` class which will manage the list.
+Great! We have nodes. Next step is to build out the `LinkedList` class.
 
-We are also going to build the `append`, `all` and `count` methods. `append` will add another node to the end of the list, `all` will return the data of the linked list in order, and `count` will return the number of nodes in the list.
+We are also going to build the `append`, `all` and `count` methods. `append` will add nodes to the end of the list, `all` will return the data of the linked list in order, and `count` will return the number of nodes in the list.
+
+Let's also make sure that we can append multiple nodes using the `append` method.
 
 Expected behavior:
 
 ```ruby
-> require "./lib/jungle_beat"
-> jb = JungleBeat.new("deep")
-=> 1 # number of nodes inserted
-> jb.head
-=> <Node data="deep" next_node=nil #4567890456789456789>
-> jb.head.data
-=> "deep"
-> jb.count
-=> 1
-> jb.append("plop")
+> require "./lib/linked_list"
+> list = LinkedList.new
+> list.append("plop pa")
```

Comment:
let's just use one word here to avoid confusing people about the single data / multi beat stuff